src/libpgmoneta/prometheus.c

```diff
@@ -2195,6 +2195,89 @@ size_information(int client_fd)
       data = NULL;
    }
 
+   data = pgmoneta_append(data, "#HELP pgmoneta_backup_newest_throughput The throughput of the newest backup for a server\n");
```

Comment:
What is "newest" for? You are getting the througput for each and every backup, aren't you?